src/analysis.zig

```diff
@@ -2542,7 +2542,7 @@ fn makeInnerScope(allocator: std.mem.Allocator, context: ScopeContext, node_idx:
         };
 
         if (container_field) |_| {
-            if (!std.mem.eql(u8, name, "_")) {
+            if (!std.mem.eql(u8, name, "_") and !std.mem.eql(u8, name, "other")) {
```

Comment:
What is the purpose of the "other" case?

Reply:
I'm not too familiar with the codebase but I reasoned the if is to weed out duplicates given the (!std.mem.eql(u8, name, "_"))
and found that a few container_field names are just "other" which cause the overwriting of previous entries and their .documentation field which led to the loss of the .documentation.value pointer value

I've updated the code to catch any dupes